cmd/list_bundles.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/opdev/opcap/internal/bundle"
+
+	"github.com/spf13/cobra"
+)
+
+var listBundlesFlags struct {
+	bundlesPath string
+}
+
+func listBundlesCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "bundles",
+		Short: "List all bundles and versions",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			bundles, err := bundle.List(listBundlesFlags.bundlesPath)
+			if err != nil {
+				return err
+			}
+
+			headings := "Package Name\tStarting CSV\tVersion\tDefault Channel\tOcpVersions"
+			w := tabwriter.NewWriter(cmd.OutOrStdout(), 0, 0, 2, ' ', 0)
+			fmt.Fprintln(w, headings)
+			for _, bundle := range bundles {
+				packageInfo := []string{bundle.StartingCSV, bundle.PackageName, bundle.Channel, bundle.Version, bundle.OcpVersions}
+				fmt.Fprintln(w, strings.Join(packageInfo, "\t"))
+			}
+			w.Flush()
+
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVar(&listBundlesFlags.bundlesPath, "bundles-path", "internal/bundle/.test/repo/certified-operators/operators",
```

Comment:
Should this be a directory? Or a git repo? Or support both?

Reply:
Directory. Name changed.